src/pages/Dashboard.tsx

```diff
@@ -1,35 +1,188 @@
+            takeaways: 'string string string string string string string string',
+            requests: ['RequestId1'],
+            collaborators: ['UserId1']
+        };
+        try {
+            const insightId: string = await addInsight(TEST_ORGANIZATION, TEST_GROUP, hardcodedInsight);
+            displayedInsights[insightId] = hardcodedInsight;
+            setDisplayedInsights({...displayedInsights});
+        } catch (error) {
+            // Replace with user visible messaging
+            console.log("Could not create new insight");
+        }
+    }
+
+    // Generate and store hardcoded request object for now
+    const createNewFiledRequest = async () => {
+        const hardcodedFiledRequest: FiledRequest = {
+            createdBy: "BigBossId",
+            createDate: 123,
+            lastUpdated: 123,
+            title: "FireBase - Title blah blah blah",
+            assignee: "CodeMonkeyId",
+            priority: "High",
+            status: "Assigned",
+            deadline: 123
+        };
+        try {
+            const filedRequestId: string = await addFiledRequest(TEST_ORGANIZATION, TEST_GROUP, hardcodedFiledRequest);
+            displayedRequests[filedRequestId] = hardcodedFiledRequest;
+            // TODO: Refactor, current state is a bit hacky to force React to update
+            setDisplayedRequests({...displayedRequests});
+        } catch (error) {
+            // Replace with user visible messaging
+            console.log("Could not create new request");
+        }
+    }
+
+    const generateDeleteFiledRequestHandler = (filedRequestId: string): Function => {
+        return async () => {
+            try {
+                await deleteInsight(TEST_ORGANIZATION, TEST_GROUP, filedRequestId);
+                delete displayedRequests[filedRequestId];
+                // TODO: Refactor, current state is a bit hacky to force React to update
+                setDisplayedRequests({...displayedRequests});
+            } catch {
+                console.log("Could not delete filed request");
+            }
+        }
+    };
+
+    const generateDeleteInsightHandler = (insightId: string): Function => {
+        return async () => {
+            try {
+                await deleteInsight(TEST_ORGANIZATION, TEST_GROUP, insightId);
+                delete displayedInsights[insightId];
+                setDisplayedInsights({...displayedInsights});
+            } catch {
+                console.log("Could not delete insight");
+            }
+        }
+    };
+
     return (
-        <TicketFeedContainer insightData={InsightData}/>
+        <Box sx={{
+            border: '1px #000 solid',
+            background: '#fff',
+            width: '100vw',
+            height:'100vh',
+            marginTop:'100px'
+        }}>
+            {/**TODO: Current button placement is very cursed, need to adjust**/}
+            <Button onClick={async () => {await createNewFiledRequest();}}>
```

Comment:
no need for asyncs, the async/await part of this will happen in createNewFiledRequest or in a handler if we create one, you can just write `onClick={createNewFiledRequest}`